### PR TITLE
[KYUUBI #4472] add session/operation methods in AdminRestApi

### PR DESCRIPTION
**Pull request by lightning-L**
### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by lightning-L** [comment]
@turboFei

File: kyuubi-server/src/test/scala/org/apache/kyuubi/server/rest/client/AdminRestApiSuite.scala [hunk 1]
```diff
@@ -84,4 +87,64 @@ class AdminRestApiSuite extends RestClientTestHelper {
     engines = adminRestApi.listEngines("spark_sql", "user", "default", "").asScala
     assert(engines.size == 0)
   }
+
+  test("list/close session") {
+    fe.be.sessionManager.openSession(
+      TProtocolVersion.findByValue(1),
```
Comment (zwangsheng) [inline kyuubi-server/src/test/scala/org/apache/kyuubi/server/rest/client/AdminRestApiSuite.scala:93]: Looks better to keep same with below test' usage `HIVE_CLI_SERVICE_PROTOCOL_V2`

**Review by zwangsheng — APPROVED** [review]
LGTM, leave some comment

File: kyuubi-server/src/test/scala/org/apache/kyuubi/server/rest/client/AdminRestApiSuite.scala (hunk 1 above)
Comment (lightning-L) [reply kyuubi-server/src/test/scala/org/apache/kyuubi/server/rest/client/AdminRestApiSuite.scala]: thanks. fixed

**Review by turboFei — APPROVED** [review]

**Review by bowenliang123 — APPROVED** [review]

**Comment by turboFei** [comment]
thanks, merging to master

**Pull request closed without merging**